.github/workflows/ci.yml

```diff
@@ -11,15 +11,38 @@ jobs:
   ci:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Setup Node.js
+      - name: "Checkout SDK"
+        uses: actions/checkout@v2
+      - name: "Prettier Check"
+        uses: creyD/prettier_action@v4.0
+        with:
+          dry: True
+          prettier_options: --config .prettierrc --check **/*.ts
+      - name: "Checkout Erdstall Core"
+        uses: actions/checkout@v2
+        with:
+          ssh-key: ${{ secrets.SSH_PRIV_KEY }}
+          repository: "perun-network/erdstall-ext"
+          path: "erdstall"
+      - name: "Set up Go 1.x"
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.17
+      - name: "Build walkthrough operator and store in environment"
+        working-directory: "erdstall"
+        run: |
+          go build -o wo ./cmd/walkthrough/main.go
+          echo "ERDSTALL_EXE=$( realpath wo )" >> $GITHUB_ENV
```

Comment:
Why so complicated? Can't you just do `ERDSTALL_EXE=erdstall/wo yarn test:integration` below?

Reply:
That will work too. I just prefer absolute paths, because of relative paths always being a security nightmare in bash scripts.

But I guess it should not be a problem here.